manifests/a/AliceTeam/CarnegieMellonUniversity/Alice3/3.7.0.0.876/AliceTeam.CarnegieMellonUniversity.Alice3.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AliceTeam.CarnegieMellonUniversity.Alice3
+PackageVersion: 3.7.0.0.876

```

```suggestion
PackageVersion: 3.7.0.0
```
Package is marketed as `3.7.0.0` in publisher's [GitHub Releases](https://github.com/TheAliceProject/alice3/releases/tag/3.7.0.0)